internal/pkg/handler/updated-handler.go

```diff
@@ -20,11 +32,216 @@ func (r ResourceUpdatedHandler) Handle() error {
 		// process resource based on its type
 		if _, ok := r.Resource.(*v1.ConfigMap); ok {
 			logrus.Infof("Performing 'Updated' action for resource of type 'configmap'")
```

Comment:
This log should be changed to something like: "Configmap: 'configmapname' updated, performing update action" and then the next log where pods are restarted can say something like this Deployment, Daemonsets or statefulset was reloaded due to change or etc